doc/mgr/administrator.rst

```diff
@@ -53,6 +53,27 @@ by a standby.
 If you want to pre-empt failover, you can explicitly mark a ceph-mgr
 daemon as failed using ``ceph mgr fail <mgr name>``.
 
+Performance
+-----------
+
+All the mgr modules share a cache that can be enabled with
+``ceph config set mgr mgr_ttl_cache_expire_seconds <seconds>``, where seconds
+is the time to live of the cached python objects.
```

Comment:
I'd first start with an explanation about the rationale of this (e.g.: "under heavy loads or large clusters (~ 1,000s OSDs) the ceph-mgr daemon might show performance issues").

And then: "In order to alleviate those/improve performance, a cache can be enabled to improve the ceph-mgr API response times"... or so.

Personally I don't think it's nice to burden Ceph users with the decision of how many seconds this needs to be cached. I'd recommend to go with a proven settings (e.g.: 5 secs).

That doesn't mean that the TTL time doesn't have to be exposed in a setting, but I'd keep that setting for devel-only level (while the other one can be advanced-level instead).